includes/class-wc-stripe-account.php

```diff
@@ -0,0 +1,159 @@
+<?php
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * WC_Stripe_Account class.
+ *
+ * Communicates with Stripe API.
+ */
+class WC_Stripe_Account {
+
+	const ACCOUNT_OPTION = 'wcstripe_account_data';
+	const ACCOUNT_API    = 'account';
+
+	/**
+	 * Gets and caches the data for the account connected to this site.
+	 *
+	 * @return array Account data or empty if failed to retrieve account data.
+	 */
+	public function get_cached_account_data() {
```

Comment:
What are your thoughts on having 2 separate transient keys: one for "live" and the other for "test" mode.

That way you don't need to add the checks on the `read_account_from_cache` or `cache_account` methods - the transient key could be "generated" in a method in this class (e.g.: get_transient_key()`) for both `read_account_from_cache` and `cache_account`.